dmoj/celery.py

```diff
@@ -23,5 +24,4 @@
 
 @task_failure.connect()
 def celery_failure_log(sender, task_id, exception, traceback, *args, **kwargs):
-    logger.error('Celery Task %s: %s on %s', sender.name, task_id, socket.gethostname(),  # noqa: G201
-                 exc_info=(type(exception), exception, traceback))
+    logger.exception('Celery Task %s: %s on %s', sender.name, task_id, socket.gethostname())  # noqa: G201
```

Comment:
Pretty sure this is wrong since it discards the exception we are given. Please revert and put the `# noqa` comment in the correct place.

Reply:
this was raised by `flake8-logging-format`

Reply:
```(dmojsite) riolku@midas:~/workspace/site/site$ flake8
./dmoj/celery.py:27:5: G201 Logging: .exception(...) should be used instead of .error(..., exc_info=True)
(dmojsite) riolku@midas:~/workspace/site/site$ flake8
./dmoj/celery.py:27:5: G202 Logging statement has redundant exc_info
```

Reply:
Oh. I see. Let me fix that.

Reply:
Had to move `# noqa` around to avoid the linter error, because of black's formatting.